### DISCUSSION TITLE
Whitelist/blacklist repositories

### DISCUSSION THREAD

**Discussion by irishismyname** [containers/podman#21309] [General]
We're trying to figure out how to make images from the web _reasonably_ safe to use for our developers without "mirroring the internet" and restricting them to a locally hosted repository. Could there be some way to whitelist certain repositories like `docker.io/python:*` (from Docker hub) or `mcr.microsoft.com/powershell:*` with podman?

**Comment by rhatdan** [comment]
Take a look at `podman image trust`

**Reply by irishismyname** [reply]
😮 thank you!
